src/applications/personalization/dashboard/tests/e2e/appointments-v2-errors.cypress.spec.js

```diff
@@ -0,0 +1,164 @@
+
+describe('MyVA Dashboard - Appointments v2 Error States', () => {
+  beforeEach(() => {
+    cy.intercept('/v0/profile/service_history', serviceHistory);
+    cy.intercept('/v0/profile/full_name', fullName);
+    cy.intercept(
+      '/v0/disability_compensation_form/rating_info',
+      disabilityRating,
+    );
+    cy.intercept('/v0/evss_claims_async', claimsSuccess());
+    cy.intercept('/v0/appeals', appealsSuccess());
+    cy.intercept(
+      '/v0/disability_compensation_form/rating_info',
+      disabilityRating,
+    );
+    cy.intercept('/v1/facilities/va?ids=*', MOCK_FACILITIES);
+    cy.intercept('GET', '/data/cms/vamc-ehr.json', vamcErc);
+    cy.intercept('GET', '/v0/feature_toggles*', {
+      data: {
+        type: 'feature_toggles',
+        features: [
+          {
+            name: featureFlagNames.showMyVADashboardV2,
+            value: true,
+          },
+        ],
+      },
+    });
+  });
+
+  describe('when the user has VA Healthcare', () => {
+    context('when there is a 500 error fetching VA appointments', () => {
+      it('should show the appointments error alert and never call the facilities API', () => {
+        const getFacilitiesStub = cy.stub();
+        spyOnFacilitiesEndpoint(getFacilitiesStub);
+        cy.intercept('GET', '/vaos/v2/appointments*', {
+          statusCode: 500,
+          body: ERROR_500,
+        });
+
+        cy.login(mockUser);
+        cy.visit('my-va/');
+
+        cy.findByText(alertText).should('exist');
+        cy.should(() => {
+          expect(getFacilitiesStub).not.to.be.called;
+        });
+        cy.findByTestId('view-manage-appointments-link-from-error').should(
+          'exist',
+        );
+        cy.findByTestId('view-manage-appointments-link-from-cta').should(
+          'not.exist',
+        );
+        cy.findByTestId('view-your-messages-link-from-cta').should('exist');
+        cy.findByTestId('refill-prescriptions-link-from-cta').should('exist');
+        cy.findByTestId('request-travel-reimbursement-link-from-cta').should(
+          'exist',
+        );
+        cy.findByTestId('get-medical-records-link-from-cta').should('exist');
+        cy.findByTestId('apply-va-healthcare-link-from-cta').should(
+          'not.exist',
+        );
+
+        cy.injectAxeThenAxeCheck();
+      });
+    });
+
+    context('when there is a 400 error fetching VA appointments', () => {
+      it('should show the appointments error alert and never call the facilities API', () => {
+        const getFacilitiesStub = cy.stub();
+        spyOnFacilitiesEndpoint(getFacilitiesStub);
+        cy.intercept('GET', '/vaos/v2/appointments*', {
+          statusCode: 400,
+          body: ERROR_400,
+        });
+
+        cy.login(mockUser);
+        cy.visit('my-va/');
+
+        cy.findByText(alertText).should('exist');
+        cy.should(() => {
+          expect(getFacilitiesStub).not.to.be.called;
+        });
```

Comment:
I'm curious about this assertion. I've never used .should directly off `cy` and the docs say that it shouldn't be used this way

https://docs.cypress.io/api/commands/should

I think it's fine for now, but was this something that was copied from the old error e2e test?

Reply:
Yea, go ahead and remove it